app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -5,28 +5,19 @@ const toggleWaiting = () => {
   document.getElementById('spinner').classList.toggle('hidden')
   document.getElementById('error-summary').classList.add('hidden')
 
-  var paymentMethodSubmitElement = document.getElementById('apple-pay-payment-method-submit')
-  if (typeof paymentMethodSubmitElement !== 'undefined' && paymentMethodSubmitElement !== null) {
-    paymentMethodSubmitElement.classList.add('hidden')
+  var paymentDetailsHeader = document.querySelector('.govuk-heading-l.web-payment-button-heading')
+  if (typeof paymentDetailsHeader !== 'undefined' && paymentDetailsHeader !== null) {
+    paymentDetailsHeader.style.display = "none"
   }
 
-  paymentMethodSubmitElement = document.getElementById('google-pay-payment-method-submit')
-  if (typeof paymentMethodSubmitElement !== 'undefined' && paymentMethodSubmitElement !== null) {
-    paymentMethodSubmitElement.classList.add('hidden')
+  var webPaymentContainer = document.querySelector('.apple-pay-container')
+  if (typeof webPaymentContainer !== 'undefined' && webPaymentContainer !== null) {
+    webPaymentContainer.style.display = "none"
   }
 
-  var paymentMethodDivider = document.getElementById('apple-pay-payment-method-divider')
-  if (typeof paymentMethodDivider !== 'undefined' && paymentMethodDivider !== null) {
-    paymentMethodDivider.classList.add('hidden')
-    paymentMethodDivider.classList.remove('pay-divider')
-    document.getElementById('apple-pay-payment-method-divider-word').classList.add('hidden')
-  }
-
-  paymentMethodDivider = document.getElementById('google-pay-payment-method-divider')
-  if (typeof paymentMethodDivider !== 'undefined' && paymentMethodDivider !== null) {
-    paymentMethodDivider.classList.add('hidden')
-    paymentMethodDivider.classList.remove('pay-divider')
-    document.getElementById('google-pay-payment-method-divider-word').classList.add('hidden')
+  webPaymentContainer = document.querySelector('.google-pay-container')
```

Comment:
I think we should give this a different name rather than reassigning the existing variable.

Reply:
Done.